### PR TITLE
UserService: create new postman tests collection

### PR DESCRIPTION
**Pull request by Dimentr01**

### CODE REVIEW

**Comment by nmamro** [comment]
Do we integrate postman tests here or should we keep them in postman? I just don't really understand that now.

**Comment by Dimentr01** [comment]
> Do we integrate postman tests here or should we keep them in postman? I just don't really understand that now.

we keep all tests in postman's collection

**Review by nmamro — APPROVED** [review]

File: Services/UserService/Startup.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseHttpsRedirection();
 
             app.UseRouting();
-            app.UseMiddleware<TokenMiddleware>();
+            //app.UseMiddleware<TokenMiddleware>();
```
Comment (ifelseelif) [inline Services/UserService/Startup.cs]: I think that here it is necessary to add TODO because antoher person can forget about this comment.

**Comment by nmamro** [comment]
Since we decided to edit folders' structure in a Poll, it's better to do it here

File: Services/UserService/Startup.cs (hunk 1 above)
Comment (Dimentr01) [reply Services/UserService/Startup.cs]: > I think that here it is necessary to add TODO because antoher person can forget about this comment.

maybe

Comment (Dimentr01) [reply Services/UserService/Startup.cs]: > I think that here it is necessary to add TODO because antoher person can forget about this comment.

I did

**Review by ifelseelif — APPROVED** [review]

**Review by Cerberous007 — APPROVED** [review]

**Review by DanilFadeeev — APPROVED** [review]

**Review by studokim — APPROVED** [review]

**Pull request merged**